cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/StageLifecycle.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.templates.etl.api;
+
+/**
+ * Provides lifecycle methods for ETL Pipeline Stages.
+ */
+public interface StageLifecycle {
+
+  /**
+   * Destroy the Pipeline Stage.
+   */
+  void destroy();
```

Comment:
This interface is exactly the same as `ProgramLifecycle` in cdap-api. Why not just use that one?